Java 25

vmikhailenko opened the pull request:
Uses Java 25.

vmikhailenko committed:
Java 25

davidhunglam committed:
Merge branch 'main' into java-use-25

davidhunglam commented on pom.xml:
```diff
@@ -22,8 +22,8 @@
-    <jdk.version>21</jdk.version>
-    <cds.services.version>4.0.2</cds.services.version>
+    <jdk.version>25</jdk.version>
+    <cds.services.version>4.3.1</cds.services.version>
```
Should we use the latest version?

davidhunglam committed:
fix: increase karma browserNoActivityTimeout to prevent CI disconnect

The OPA5 integration tests exceed the default 30s browserNoActivityTimeout
when running against the Java server in CI, causing Chrome Headless to
disconnect before tests complete. Increase timeout to 60s.

davidhunglam commented:
Hi @chgeo, could you approve this for us? I'm the last CAP Java member left that has access to this repo, but I can't merge it with my own approval as I've made the last change...
